composer.json

```diff
@@ -1,79 +1,85 @@
 {
     "name": "zendframework/zendframework",
-    "description": "Zend Framework 2",
-    "type": "library",
+    "description": "Zend Framework",
+    "type": "metapackage",
```

Comment:
Nope - just confirmed that when you install a metapackage, nothing related to the package is installed in the `vendor/` directory — only the packages listed as requirements are installed. As such, the end-user will have no files from this repository installed at all. :smile: